spec/unit/matrix-user.spec.js

```diff
@@ -0,0 +1,35 @@
+const MatrixUser = require("../../lib/models/users/matrix");
+
+describe("MatrixUser", function() {
+    describe("escapeUserId", function() {
+        it("should not escape acceptable userids", function() {
```

Comment:
Please keep `user` and `ids` as two words, preferably also "IDs". Same applies to below.